config/environments/test.rb

```diff
@@ -8,53 +6,48 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
-  config.cache_classes = false
-  config.action_view.cache_template_loading = true
+  # While tests run files are not watched, reloading is not necessary.
+  config.enable_reloading = false
 
-  # Eager loading loads your whole application. When running a single test locally,
-  # this probably isn't necessary. It's a good idea to do in a continuous integration
-  # system, or in some way before deploying your code.
+  # Eager loading loads your entire application. When running a single test locally,
+  # this is usually not necessary, and can slow down your test suite. However, it's
+  # recommended that you enable it in continuous integration systems to ensure eager
+  # loading is working properly before deploying your code.
   config.eager_load = ENV["CI"].present?
 
-  # Configure public file server for tests with Cache-Control for performance.
-  config.public_file_server.enabled = true
-  config.public_file_server.headers = {
-    "Cache-Control" => "public, max-age=#{1.hour.to_i}",
-  }
+  # Configure public file server for tests with cache-control for performance.
+  config.public_file_server.headers = { "cache-control" => "public, max-age=3600" }
```

Comment:
```ruby
config.public_file_server.headers = { "cache-control" => "public, max-age=#{1.hour.to_i}" }
```

Is easier to read for a dev in a hurry!

Reply:
rails app:update removed this. Can do by adding `require "active_support/core_ext/integer/time"` back to the top but performance wise this would get loaded on every test run, slowing it down a [teeny] bit. Will add back in for now as we do not work on it much and is consistent with development and production.

Reply:
Happy to leave 3600 if speed is a concern, maybe chuck a comment after with `#3600 = 1 hour`
I know I always have to pause at seconds to longer conversions...
>Hmmm, 3600 I think that's an hour? ... googles ... yes!

Reply:
I will add back, performance is not a real concern for this app.